include/GenSync/Syncs/IBLTSync_Adaptive_PartialDecode.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * The {@link IBLTSync_Adaptive_PartialDecode} method syncs with another {@link IBLTSync_Adaptive_PartialDecode} method by sending an IBLT containing
+ * its set. Upon receiving this IBLT, the server performs a subtract operation on both IBLTs
+ * and uses the resulting IBLT to calculate the symmetric set difference. These differences
+ * are then sent back to the client. If the decoding process failed, we construct an new IBLT.
+ * The new IBLT is built with doubled size minus the number of peeled elements and we do not insert peeled elements.
```

Comment:
Maybe a note that doubling in size is the optimal strategy with respect to overhead over the "clairvoyant" IBLT?